docs/reference/glossary.md

```diff
@@ -123,6 +128,22 @@ The editing interface for this join allows the user to pick offices to associate
 <!-- See the [apostrophe-schemas](/reference/modules/apostrophe-schemas/README.md) module documentation for more information about joins. -->
 
 <!-- TODO: Display this when other locale information is added. -->
-<!-- ## Document set
+<!-- ### Document set
+
+The database documents that exist as mode and locale versions of one another. For example, in a project with an English and a French locale, the home page would be managed in the database by six documents: 1) the draft English, 2) published English, 3) previous English, 4) draft French, 5) published French, and 6) previously published French. These six database documents make up the document set. -->
```

Comment:
previously published English (consistency)

Reply:
You could mention that they all share an `aposDocId`, which is also a prefix of the `_id` of each document.

Reply:
this bit isn't actually new (or being published. though this might be the time to revisit it.